api-gateway/src/main/resources/static/scripts/bill-history/bill-history.controller.js

```diff
@@ -6,6 +6,26 @@ angular.module('billHistory')
         let self = this;
         self.billHistory = []
 
+        self.owners = [
```

Comment:
If hardcoding was the only way to solve this problem, maybe we should consider using an aggregator